frontend/base/src/main/java/com/virtuslab/gitmachete/frontend/resourcebundles/GitMacheteBundle.java

```diff
@@ -26,12 +28,21 @@ private GitMacheteBundle() {}
    *             but each parameter must be a non-null {@link String} and not just a nullable {@link Object}
    * @return the formatted string
    */
-  public static String format(@I18nFormatFor("#2") String format, String @MinLen(1)... args) {
+  public static @PolyTainted String format(@PolyTainted @I18nFormatFor("#2") String format, String @MinLen(1)... args) {
     return MessageFormat.format(format, (@Nullable Object[]) args);
   }
 
   @I18nMakeFormat
   public static String getString(@PropertyKey(resourceBundle = BUNDLE) String key) {
     return instance.getString(key);
   }
+
+  @I18nMakeFormat
+  public static @Untainted String getNonHtmlString(@PropertyKey(resourceBundle = BUNDLE) String key) {
+    return instance.getString(key);
+  }
+
+  public static @Untainted @SuppressWarnings("regexp") String changeNewLineSign(@Untainted String text) {
```

Comment:
If this method stays (and I'd strongly recommend to remove it and pass the newline by other means), pls rename to
```suggestion
  public static @Untainted @SuppressWarnings("regexp") String replaceHtmlLineBreakWithNewLineCharacter(@Untainted String text) {
```
for clarity

Reply:
Btw yeah... `text` is `@Untainted` but might contain a `<br/>`... something is _not yes_ here 😅

Reply:
I'm taking this prefix to work on, because it actually looks bad (all because of the unfortunate `fetchNotificationIfFailed`). I will try to come up with some way to systematize it